index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="style.css">
+        <title>finish-mobile-version</title>
+    </head>
+<body>
+    <div class="grid-container">
+        <div class="flex1">
+            <div class="snapshoot">
+                    <img src="./images/Snapshoot Portfolio.png"/>                
+            </div>
+            <div class="tonic">Tonic</div>
+            <div>3</div>
+            <div clas=""><p>A daily selection of privately </br>personalized reads; no accounts or </br>sign-ups required.</p></div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex2">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex3">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+        <div class="flex4">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+        </div>
+    </div>
```

Comment:
Good job using GRID CONTAINER. However, you need to finish your code. in unclear in this time.